policykit/integrations/slack/models.py

```diff
@@ -35,20 +35,16 @@ class SlackCommunity(Community):
     bot_id = models.CharField('bot_id', max_length=150, unique=True, default='')
 
     def notify_action(self, action, policy, users, post_type='channel', template=None, channel=None):
-        from slackintegration.views import post_policy
+        from intergrations.slack.views import post_policy
```

Comment:
just checking is this a typo? Shouldn't it be "integrations"

Reply:
🤦‍♀️  thank you